src/components/nodes-table/nodes-table.js

```diff
@@ -37,6 +38,31 @@ export default {
         lastActives: { type: 'date' }
       }
     },
+    initSorting () {
+      return {
+        by: ['node_id'],
+        desc: [false]
+      }
+    },
+    loadObject (key, defaultValue) {
+      const filtersStr = localStorage.getItem(key)
+      let filters
+      try {
+        filters = JSON.parse(filtersStr)
+      } catch (e) {
+        filters = undefined
+      }
+      if (
+        !filters ||
+        (Object.keys(filters).length === 0 && filters.constructor === Object)
+      ) {
+        filters = defaultValue
+      }
+      return filters
+    },
+    storeObject (key, val) {
+      localStorage.setItem(key, JSON.stringify(val))
```

Comment:
you could make load and store more generic by checking the type of val like `typeof val === 'string' ? val : JSON.stringify(val)` so you can use the same functions for `node_itemsPerPage` local storage key

Reply:
Yes, I already thought about that :-).
I'll adjust the code and apply it to itemsPerPage too.

Reply:
@robertsLando Nevermind - the refactoring is already done and I now the settings 'nodes_showHidden' and 'dark' use the Settings module too.